package.json

```diff
@@ -36,12 +36,12 @@
     "ES2016"
   ],
   "dependencies": {
-    "babel-runtime": "5.6.20",
-    "bluebird": "^2.9.33",
+    "babel-runtime": "^5.8.29",
```

Comment:
In other words, better not touch the babel-runtime version here as it will be overwritten before we release a new version of Harmonic anyway.